build/ci/net7/global.json

```diff
@@ -0,0 +1,14 @@
+{
+	"sdk": {
+		"version": "7.0.406",
+		"allowPrerelease": false,
+		"rollForward": "latestFeature"
```

Comment:
Do we want to set this to "disable" for CI stability?

Reply:
That sounds to me like a good idea

Reply:
Will do the change once I get confirmation from @jeromelaban :)

Reply:
Yes, let's freeze to a specific version, that is until we get https://github.com/dotnet/sdk/issues/30103 integrated.